ironfish-rust-nodejs/tests/demo.test.slow.ts

```diff
@@ -59,12 +59,12 @@ describe('Demonstrate the Sapling API', () => {
 
     const decryptedNoteBuffer = encryptedNote.decryptNoteForOwner(key.incoming_view_key)
     expect(decryptedNoteBuffer).toBeInstanceOf(Buffer)
-    expect(decryptedNoteBuffer.byteLength).toBe(115)
+    expect(decryptedNoteBuffer!.byteLength).toBe(115)
```

Comment:
These lines were complaining about possibly being null, so using the not-null assertion operator to silence those